support/behandeling-queries.js

```diff
@@ -0,0 +1,273 @@
+      ${prefixMap.get("besluit").toSparqlString()}
+      ${prefixMap.get("dct").toSparqlString()}
+      ${prefixMap.get("ext").toSparqlString()}
+      ${prefixMap.get("pav").toSparqlString()}
+      ${prefixMap.get("schema").toSparqlString()}
+      ${prefixMap.get("mu").toSparqlString()}
+      SELECT * 
+      WHERE {
+        BIND (<${uri}> AS ?agendaUri)
+        <${uri}> besluit:geplandOpenbaar ?geplandOpenbaar.
+        <${uri}> dct:title ?titel.
+        ${sparqlEscapeUri(uri)} schema:position ?position.
+        ?bva dct:subject ${sparqlEscapeUri(uri)}.
+        ?bva mu:uuid ?bvaUuid.
+        ?bva ext:hasDocumentContainer ?document.
+        ?bva besluit:openbaar ?openbaar.
+        ?document pav:hasCurrentVersion ?editorDocument.
+        ?editorDocument <http://mu.semte.ch/vocabularies/core/uuid> ?editorDocumentUuid;
+          ext:editorDocumentContent ?documentContent.
+        OPTIONAL {
+          ?bva besluit:heeftSecretaris ?secretaris.
+          ?bva besluit:heeftVoorzitter ?voorzitter.
+        }
+      }
+    `)
+    const agendapunten = queryResults.results.bindings[0];
+    const mandateesResults = await query(`
+    ${prefixMap.get("besluit").toSparqlString()}
+    ${prefixMap.get("mandaat").toSparqlString()}
+    ${prefixMap.get("foaf").toSparqlString()}
+    ${prefixMap.get("persoon").toSparqlString()}
+      SELECT * WHERE {
+        ${sparqlEscapeUri(agendapunten.bva.value)} besluit:heeftAanwezige ?mandatarisUri.
+        ?mandatarisUri mandaat:isBestuurlijkeAliasVan ?personUri.
+        ?personUri foaf:familyName ?familyName.
+        ?personUri persoon:gebruikteVoornaam ?name.
+      }
+    `)
+    const presentMandatees = mandateesResults.results.bindings.map(mandatee => ({
+      uri: mandatee.mandatarisUri.value,
+      personUri: mandatee.personUri.value,
+      name: mandatee.name.value,
+      familyName: mandatee.familyName.value
+    }));
+    const stemmings = await fetchStemmings(agendapunten.bva.value);
+    return {
+      uri: agendapunten.agendaUri.value,
+      geplandOpenbaar: agendapunten.geplandOpenbaar.value,
+      position: agendapunten.position.value,
+      titel: agendapunten.titel.value,
+      behandeling: {
+        uri: agendapunten.bva.value,
+        uuid: agendapunten.bvaUuid.value,
+        openbaar: agendapunten.openbaar.value,
+        presentMandatees,
+        stemmings,
+        document: {
+          uuid: agendapunten.editorDocumentUuid.value,
+          content: agendapunten.documentContent.value
+        }
+      }
+    }
+  });
+  /** @type {Support.QueryResult<"agendaUri" | "geplandOpenbaar" | "titel">[]} */
+  const agendapunten = await Promise.all(agendaQueries);
+
+  const agendapuntenSorted = agendapunten.sort((a, b) => a.position > b.position ? 1 : -1)
+
+
+  return {
+    bestuursorgaan: bestuursorgaan.value,
+    geplandeStart: geplandeStart.value,
+    uri: uri.value,
+    agendapunten: agendapuntenSorted,
+  };
+}
+
+async function fetchStemmings(bvaUri) {
```

Comment:
Doesn't matter that much, but the plural for `stemming` is `stemmingen`

Reply:
Thanks, I will change this